Tests/UnsignedVarInt.lean

```diff
@@ -24,5 +24,5 @@ def cases : List Case :=
   , Case.mk 16384 { data := #[0b10000000, 0b10000000, 0b000000001] }
   ]
 
-def main := lspec "UnsignedVarInt works properly" $
-  it "converts Nat ↔ UnsignedVarInt properly" (findFailing cases) shouldBeEmpty
+def main := lspec do 
+  test  "UnsignedVarInt works properly" (findFailing cases).isEmpty
```

Comment:
```suggestion
  test  "converts Nat ↔ UnsignedVarInt properly" (findFailing cases).isEmpty
```
Let's keep the description for specific tests and drop the generic description for the test suite